In perl 5.26, there is no current directory in library loading path anymore.
This makes many modules fail to build.

Error:
  | Can't locate inc:: ... in @INC (you may need to install the ... module)
  | Can't locate t:: ... in @INC (you may need to install the ... module)
  | do "foo.pl" failed, '.' is no longer in @INC; did you mean do "./foo.pl"?

diff --git a/write_buildcustomize.pl b/write_buildcustomize.pl
index 8666a6b..6b4de3a 100644
--- a/write_buildcustomize.pl
+++ b/write_buildcustomize.pl
@@ -3,7 +3,7 @@
 use strict;
 
 my $osname = $^O;
-my $file = 'lib/buildcustomize.pl';
+my $file = './lib/buildcustomize.pl';
 
 if ( @ARGV % 2 ) {
     my $dir = shift;
